suite/runsuite_wrapper.pl

```diff
@@ -231,6 +231,48 @@
+                'code_api,thread_private|common.decode-stress' => 1, # i#4845
+                'code_api,thread_private,disable_traces|common.decode-stress' => 1, # i#4845
+                'prof_pcs|common.broadfun' => 1, # i#4845
+                'prof_pcs|common.fib' => 1, # i#4845
+                'prof_pcs|common.decode-bad' => 1, # i#4845
+                'prof_pcs|common.decode' => 1, # i#4845
+                'prof_pcs|common.decode-stress' => 1, # i#4845
+                'prof_pcs|common.eflags' => 1, # i#4845
+                'prof_pcs|common.floatpc' => 1, # i#4845
+                'prof_pcs|common.floatpc_xl8all' => 1, # i#4845
+                'prof_pcs|common.getretaddr' => 1, # i#4845
+                'prof_pcs|common.nativeexec' => 1, # i#4845
+                'prof_pcs|common.protect-dstack' => 1, # i#4845
+                'prof_pcs|common.segfault' => 1, # i#4845
+                'prof_pcs,thread_private|common.broadfun' => 1, # i#4845
+                'prof_pcs,thread_private|common.fib' => 1, # i#4845
+                'prof_pcs,thread_private|common.decode-bad' => 1, # i#4845
+                'prof_pcs,thread_private|common.decode' => 1, # i#4845
+                'prof_pcs,thread_private|common.decode-stress' => 1, # i#4845
+                'prof_pcs,thread_private|common.eflags' => 1, # i#4845
+                'prof_pcs,thread_private|common.floatpc' => 1, # i#4845
+                'prof_pcs,thread_private|common.floatpc_xl8all' => 1, # i#4845
+                'prof_pcs,thread_private|common.getretaddr' => 1, # i#4845
+                'prof_pcs,thread_private|common.nativeexec' => 1, # i#4845
+                'prof_pcs,thread_private|common.protect-dstack' => 1, # i#4845
+                'prof_pcs,thread_private|common.segfault' => 1, # i#4845
+                'code_api,thread_private,tracedump_binary|common.decode-stress' => 1, # i#4845
+                'desktop,coarse_freeze_at_exit|common.broadfun' => 1, # i#4845
+                'desktop,coarse_freeze_at_exit|common.logstderr' => 1, # i#4845
+                'desktop,coarse_freeze_at_exit|common.fib' => 1, # i#4845
+                'desktop,coarse_freeze_at_exit|common.decode-bad' => 1, # i#4845
+                'desktop,coarse_freeze_at_exit|common.decode' => 1, # i#4845
+                'desktop,coarse_freeze_at_exit|common.decode-stress' => 1, # i#4845
+                'desktop,coarse_freeze_at_exit|common.eflags' => 1, # i#4845
+                'desktop,coarse_freeze_at_exit|common.floatpc' => 1, # i#4845
+                'desktop,coarse_freeze_at_exit|common.floatpc_xl8all' => 1, # i#4845
+                'desktop,coarse_freeze_at_exit|common.getretaddr' => 1, # i#4845
+                'desktop,coarse_freeze_at_exit|common.nativeexec' => 1, # i#4845
+                'desktop,coarse_freeze_at_exit|common.protect-dstack' => 1, # i#4845
+                'desktop,coarse_freeze_at_exit|common.segfault' => 1, # i#4845
```

Comment:
This again looks like every single test (probably would all the non-common ones too if the #1807 limit were lifted) and so seems like it should b dealt with at a higher level than the flaky test list?

Reply:
Done.